src/sentry/south_migrations/0288_populate_release_project_new_groups.py

```diff
@@ -0,0 +1,721 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        db.commit_transaction()
+        for release in orm.Release.objects.all():
+          projects = release.projects.values_list('id', flat=True)
+          if len(projects) > 1:
+            # do something fancy where we look at Group.first_release
+            # to calculate ReleaseProject.new_group
+            for p_id in projects:
+                new_groups = orm.Group.objects.filter(
+                    first_release=release,
+                    project_id=p_id
+                ).count()
+                orm.ReleaseProject.objects.filter(
+                    release_id=release.id,
+                    project_id=p_id
+                ).update(new_groups=new_groups)
+          else:
+            # copy Release.new_groups to ReleaseProject.new_group
+            orm.ReleaseProject.objects.filter(
+                release_id=release.id,
+                project_id=projects.first()
```

Comment:
This would be incurring yet another query. But if we've coerced it to a `list()` we should be able to just do `projects[0]`.